src/twisted/conch/test/test_keys.py

```diff
@@ -861,6 +910,33 @@ def test_fromBlobECDSA(self):
         self.assertTrue(eckey.isPublic())
         self.assertEqual(ecPublicData, eckey.data())
 
+    def test_fromBlobSKECDSA(self):
+        """
+        A public SK-ECDSA key is correctly generated from a public key blob.
```

Comment:
The key is generated or loaded ?

Also, what does `correctly` means in the contest of this test ?

Minor comment. See
https://jml.io/test-docstrings/

```suggestion
        It can load public SK-ECDSA key from blob format.
```

Instead of `corretly` maybe it would help to provide a reference the the RFC or the document containg the specification for the blob format for secure keys